### ISSUE TITLE
Problem with ember.variant in configuration files

### ISSUE THREAD

**Issue by Draiken** [emberjs/ember-rails#98]
So, I recently upgraded ember from an older version, and had to add the `config.ember.variant = :production` to my `config/environments/production.rb`

The problem is, when precompiling the assets, everything goes fine, but when I start the server, I get this error:   

'method_missing': undefined method `ember' for #Rails::Application::Configuration:0x00000003e46710 (NoMethodError) 

https://gist.github.com/3793642 (stack trace)

So I remove the variant line, and the app boots normally.

But with it removed, I get errors when precompiling the assets.
I am using ember-rails 0.7.0

**Comment by avinmathew** [comment]
I recently came across the same error on a new environment. For some reason the Railtie in ember_rails.rb` was never reached. When I traced all the `require` calls, it was crashing out around `require 'execjs'` called from the `barber` gem. I then realised that I hadn't installed nodejs on that machine. It seems like the error from `execjs` caused the process to exit, but Rails still continued to load.
